pkg/server/metrics/metrics.go

```diff
@@ -164,6 +167,16 @@ func (a *ServerMetrics) SetBackendCount(count int) {
 	a.backend.WithLabelValues().Set(float64(count))
 }
 
+// BackendCountInc increments a new backend connection.
+func (a *ServerMetrics) BackendCountInc(manager string, idType string) {
```

Comment:
we have only one managed supported by the server once, so I guess it's not necessary to record metrics for each backend manager. the introduction of `idType` is valuable, but for current use cases, I'd suggest we keep the code as it is now.

Reply:
multiple backend managers are supported for one proxy-sever, close this comment

Reply:
anything else I need to do about this issue?